node/resolvers/search/index.ts

```diff
@@ -552,6 +553,9 @@ export const queries = {
 
     const sellers = await getSellers(vbase, checkout, segment?.channel, segment?.regionId)
 
+    const workspaceSearchParams = await getWorkspaceSearchParamsFromStorage(ctx)
+    console.log("supsude1", workspaceSearchParams)
```

Comment:
if you need to check this info with a console.log and it was not an accident you can use `const isLinked = process.env.VTEX_APP_LINK` to check if it is linked as we do here https://github.com/vtex-apps/search-resolver/blob/v1.x/node/clients/biggy-search.ts#L286.